languages/javascript/exercises/concept/arrays/suggestion.arrays.spec.js

```diff
@@ -0,0 +1,140 @@
+import {
```

Comment:
What is this file :) ?

Reply:
actually this is suggestion of the testcase that i make. what i'm trying to make here is to check that the student mutate the array not creating the new array @SleeplessByte

Reply:
You don't need to test for that.  Immutability / Mutability will be handled in a different exercise :)

Reply:
okay then let just delete this file : )